tests/test_gpu_common.py

```diff
@@ -19,9 +19,10 @@ def test_get_gpu_info(self):
         info = get_gpu_info()
         try:
             assert info['architecture'].lower() in ['tesla', 'geforce', 'unspecified']
-        except TypeError:
+        except KeyError:
```

Comment:
Here we need to define what we want.. let pass only these values or even empty info (which was previous behavior)

Reply:
yeah I think the current behaviour is OK